libs/vkd3d/resource.c

```diff
@@ -2401,6 +2426,30 @@ HRESULT d3d12_resource_create_committed(struct d3d12_device *device, const D3D12
             hr = hresult_from_vk_result(vr);
             goto fail;
         }
+
+        if (vkd3d_resource_can_be_vrs(device, heap_properties, desc))
```

Comment:
Are placed VRS images banned?

Reply:
Just classic D3D12 derpiness when ClearUAV requires two sets of descriptors set up, but VRS just uses the plain resource without an SRV :V

Reply:
Ah, no. I think I missed the view setup on the placed path.